tests/unit/StatusByWorkboardDispatcherTest.php

```diff
@@ -0,0 +1,119 @@
+<?php
+
+use Phragile\PhabricatorAPI;
+use Phragile\ProjectColumnRepository;
+use Phragile\StatusByWorkboardDispatcher;
+use Phragile\TransactionList;
+
+/**
+ * @covers Phragile\StatusByWorkboardDispatcher
+ */
+class StatusByWorkboardDispatcherTest extends TestCase {
+
+	private $workboardColumns = [
+		'PHID-123abc' => 'backlog',
+		'PHID-321cba' => 'done',
+		'PHID-abc123' => 'to do',
+	];
+
+	private $columnPHIDs = [
+		'backlog' => 'PHID-123abc',
+		'done' => 'PHID-321cba',
+		'to do' => 'PHID-abc123',
+	];
+
+	private function newPhabricatorAPI()
+	{
+		$phabricatorAPI = $this->getMockBuilder( PhabricatorAPI::class )
+			->disableOriginalConstructor()
+			->getMock();
+		$phabricatorAPI->method('queryPHIDs')->will($this->returnCallback(function()
+		{
+			return array_map(function($column)
+			{
+				return ['name' => $column];
+			}, $this->workboardColumns);
```

Comment:
True, I had that before like:
```php
return array_map(function($column)
{
	return ['name' => $column];
}, array_flip($this->workboardColumns));
```
but looking at this made me look like Jeroen's "double space OMG" cat.
But it doesn't really read that bad, so I'd probably change it the way you suggested.